t04_refactored.py

```diff
@@ -194,9 +194,46 @@ def team_13_adv():
 
 ###################################################################################
 
+
 def team_14_adv():
-    pass
-    # TODO Add your code here
+    # TEAM 14
+    """
+    https://docs.google.com/document/d/1IAD1loO2xwbKq6726_JV1OeaI2pLem_Wx8FiRk3GBdo/edit
+    gavin leach
+    elijah babayemi
+    """
+
+    global isDead
+    direction = input("Which direction would you like to go? [North/South/East/West]")
+
+    if direction == "North":
+        print("You didn't die. In fact, you made it out of the cave and back into society. Yay.")
+        sleep(delay)
+
+    elif direction == "South":
+        print("You unfortunately fell off a cliff and are about to die.")
+        isDead = True
+        sleep(delay)
+
+    else:
+        print("You're still in the cave, and no closer to being out of it. Sucks to be you.")
+        sleep(delay)
+
+    if isDead:
+        numbertest = input("Pick a number 1-10 to try and survive.")
+        numbertest = float(numbertest)
```

Comment:
You probably wanted an `int` here, not a `float`. What happens if they put in `10.001`?

Reply:
i fixed it, do i issue another pull request?